src/config/process.ts

```diff
@@ -0,0 +1,3 @@
+export const githubClientID = process.env.GITHUB_ID as string;
+export const githubClientSecret = process.env.GITHUB_SECRET as string;
+export const coreBaseUrl = process.env.NEXT_PUBLIC_APP_CORE_BASE_URL ?? "http://localhost:4925";
```

Comment:
Not a big issue for now but at some point it might be good to validate that these env variables are present when the process starts like we did for core.

Reply:
addressed https://github.com/Extheoisah/hxckr-web/pull/2/commits/5a3171ee7f6b2c1590730190657c31957ec2f696

Reply:
Ideally we should check each one and throw a descriptive error but I'm not going to hold this PR up over that 😄

Reply:
Thanks. I can do that in my next pr